### PR TITLE
Add support for `--ignore-catch` and fix multi-word CLI flag bug

### LINKED ISSUE #40: Suggestion: Use ignoreCatch option from type-coverage
**Is your feature request related to a problem? Please describe.**
The `ignoreCatch ` option is not currently implemented.

**Describe the solution you'd like**
Enable the option.

**Additional context**
This feature was missing from #33 and reported by https://github.com/alexcanessa/typescript-coverage-report/issues/33#issuecomment-729946011

### PR DESCRIPTION
**Pull request by dignite**
Fixes #40 

Also fixes bug where multi-word command line options were not working. Options `ignore-files` and `ignore-unread` were read correctly from `package.json` but could not be set with cli flags.

### CODE REVIEW

File: src/bin/typescript-coverage-report.ts [hunk 1]
```diff
@@ -75,7 +74,17 @@ program
   )
   .parse(argvWithVersion(process.argv));
 
-const options = getOptions(program);
```
Comment (alexcanessa) [inline src/bin/typescript-coverage-report.ts:72]: I'm ok to remove the `getOptions` function if you want. The limit of `commander` is that unfortunately doesn't give you the `defaultValue` when you try to access it.

File: src/lib/index.ts [hunk 2]
```diff
@@ -34,9 +28,9 @@ export default async function generateCoverageReport(
   const data = await getCoverage({
     strict: options.strict,
     debug: options.debug,
-    ignoreFiles: options["ignore-files"],
-    ignoreCatch: options["ignore-catch"],
-    ignoreUnread: options["ignore-unread"],
+    ignoreFiles: options.ignoreFiles,
+    ignoreCatch: options.ignoreCatch,
+    ignoreUnread: options.ignoreUnread,
     cache: options.cache
```
Comment (alexcanessa) [inline src/lib/index.ts]: At this point might as well pass option to `getCoverage`

**Review by alexcanessa — CHANGES_REQUESTED** [review]

File: src/lib/index.ts (hunk 2 above)
Comment (alexcanessa) [reply src/lib/index.ts]: Not a show blocker anyway

**Comment by alexcanessa** [comment]
@all-contributors add @dignite for code

**Review by alexcanessa — APPROVED** [review]

File: src/lib/index.ts (hunk 2 above)
Comment (dignite) [reply src/lib/index.ts]: True, good future improvement

**Pull request merged**